pkgs/tools/security/semgrep/default.nix

```diff
@@ -0,0 +1,58 @@
+{ lib
+, fetchFromGitHub
+, callPackage
+, semgrep-core
+, buildPythonApplication
+, pythonPackages
+}:
+
+let
+  common = callPackage ./common.nix { };
```

Comment:
Could this be rephrased in a way that doesn't block overlays from overriding `common` or its attributes?

Reply:
You dont need to override common. Everything can be overriden in the package as normal with overrideAttrs (probably some nuance with python)